hms-app/src/components/input-forms/HackathonForm.tsx

```diff
@@ -19,12 +19,12 @@ type IProps = {
   hackathonId: string | null
   closeAccordion?: () => void
   reload?: () => void
-  setOpened?: (boolean: boolean) => void
+  closeEditModal?: () => void
 }
 
 function HackathonForm(props: IProps) {
   const { instance } = useMsal()
-  const { context, hackathonId, closeAccordion, reload, setOpened } = props
+  const { context, hackathonId, closeAccordion, reload, closeEditModal } = props
```

Comment:
ok so having looked at this `closeAccordion` and `closeEditModal` are used when the same thing happens.  Both are called when operation that is meant to be performed is completed successfully.

as well `reload` is always called with `closeEditModal` so the two could be combined into a single thing.

I would say that you replace all three callbacks with a single one called `onOperationSuccess`

Reply:
this sounds really good

Reply:
and now it became simpler to look into 👍